.github/workflows/test-win.yml

```diff
@@ -103,7 +103,9 @@ jobs:
         with:
           conda-remove-defaults: "true"
           auto-activate-base: false
-          installer-url: https://repo.anaconda.com/miniconda/Miniconda3-py312_25.1.1-2-Windows-x86_64.exe
```

Comment:
It seems we need to keep this change to make our Windows tests work without problems, so please revert it.